chandra_aca/tests/test_drift.py

```diff
@@ -56,10 +55,9 @@ def test_get_aca_offsets_legacy():
 @pytest.mark.parametrize("kwargs", kwargs_list)
 @pytest.mark.parametrize("env_override", [None, str(Path(__file__).parent / "data")])
```

Comment:
I thought this env override to "data" used the test model that has now been removed?

Reply:
Fixed!